hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/impl/DefaultNearCache.java

```diff
@@ -109,6 +77,14 @@ protected ScheduledFuture scheduleExpirationTask(NearCacheExecutor nearCacheExec
         }
     }
 
+    private ScheduledFuture createExpirationTask() {
```

Comment:
I renamed it to `createAndScheduleExpirationTask`, so it matches the `create...` schema of the other methods.